api/controllers/tenant_manager/tenant-add.js

```diff
@@ -14,6 +14,7 @@ var inputParams = {
    username: { string: true, required: true },
    password: { string: true, required: true },
    email: { string: { email: true }, required: true },
+   url: { string: true, required: true },
```

Comment:
try to change this to:
` url: { string: {uri: true}, required: true},`

So we make sure it is a valid uri  and not some random string.